besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -55,14 +58,30 @@ public class CliqueBesuControllerBuilder extends BesuControllerBuilder {
   private long secondsBetweenBlocks;
   private final BlockInterface blockInterface = new CliqueBlockInterface();
 
+  private Optional<BuilderApi> builderApi;
+
   @Override
   protected void prepForBuild() {
     localAddress = Util.publicKeyToAddress(nodeKey.getPublicKey());
     final CliqueConfigOptions cliqueConfig = configOptionsSupplier.get().getCliqueConfigOptions();
     final long blocksPerEpoch = cliqueConfig.getEpochLength();
     secondsBetweenBlocks = cliqueConfig.getBlockPeriodSeconds();
-
     epochManager = new EpochManager(blocksPerEpoch);
+    if (cliqueConfig.getBuilderApiEndpoint().isPresent()
+        && cliqueConfig.getProposerPubKey().isPresent()) {
+      LOG.info(
+          "Connecting to builder API endpoint: {} with proposerPubKey: {}",
+          cliqueConfig.getBuilderApiEndpoint().get(),
+          cliqueConfig.getProposerPubKey().get());
+      this.builderApi =
+          Optional.of(
+              new BuilderApi(
+                  cliqueConfig.getBuilderApiEndpoint().get(),
+                  cliqueConfig.getProposerPubKey().get()));
+    } else {
+      LOG.info("Cannot connect to builder API endpoint");
```

Comment:
Misleading message gets logged - it's not about connection.
Consider: "Builder API not configured, expected API endpoint and proposer BLS public key".